packages/storybook/stories/VideoGallery/snippets/WithHorizontalGallery.snippet.tsx

```diff
@@ -49,7 +49,11 @@ export const WithHorizontalGalleryExample: () => JSX.Element = () => {
   const containerStyle = { height: '50vh' };
   return (
     <Stack style={containerStyle}>
-      <VideoGallery localParticipant={MockLocalParticipant} remoteParticipants={MockRemoteParticipants} />
+      <VideoGallery
+        localParticipant={MockLocalParticipant}
+        remoteParticipants={MockRemoteParticipants}
+        overflowGalleryLayout={'HorizontalBottom'}
```

Comment:
Just curious. Did you need to set overflowGalleryLayout prop here because 'HorizontalBottom' should be the default. Or did you just want to be verbose with which layout will be shown?

Reply:
just to be verbose. since its optional and has a default I thought it would be best to show what happens when you do apply a control. since if you were to make the component responsive in your app you would want to use the other option to make it horizontal instead of unsetting the value.